mesh_segmenter/CMakeLists.txt

```diff
@@ -29,3 +29,15 @@ catkin_add_gtest(${PROJECT_NAME}-test test/utest.cpp)
 if(TARGET ${PROJECT_NAME}-test)
     target_link_libraries(${PROJECT_NAME}-test ${PROJECT_NAME})
 endif()
+
+#############
+## Install ##
+#############
+install(
+  DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
+
+install(
+  TARGETS mesh_segmenter
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
```

Comment:
I believe shared library targets need to be installed in both the library and runtime destinations. Typically what I've done in the past is used a single `install` command to install all libraries and executables:
```cmake
install(
  TARGETS
    target_1
    target_2
  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
)
```

Reply:
Sounds good, I'll make those changes.

Reply:
Changes have been made.